cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/HttpRequestBuilderImpl.java

```diff
@@ -129,6 +129,12 @@ public final HttpPostRequest post() {
         return new HttpPostRequestImpl();
     }
 
+    @Override
```

Comment:
I haven't seen how this gets used yet but it seems weird to me that it returns a new object without somehow including or passing along the current state.

Reply:
This is a non-static inner class and the state is in the outer class, this allows for the Fluent Interface design pattern.

Here's an example for the GET: https://github.com/HydrologicEngineeringCenter/cwms-radar-client/blob/main/cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/LevelController.java#L47